README.md

```diff
@@ -14,72 +12,97 @@
 [![total](https://cranlogs.r-pkg.org/badges/grand-total/shiny.semantic)](https://CRAN.R-project.org/package=shiny.semantic)
 <!-- badges: end -->
 
-With this library it’s easy to wrap Shiny with [Semantic UI
-components](https://github.com/Semantic-Org/Semantic-UI). Add a few
-simple lines of code and some CSS classes to give your UI a fresh,
-modern and highly interactive look.
-
-`master` branch contains the stable version. Use `develop` branch for
-latest features.
+**Fomantic (Semantic) UI wrapper for Shiny**
 
-<h2>
+With this library it is easy to wrap Shiny with **[Fomantic
+UI](https://fomantic-ui.com/)** (previously *Semantic*). Add a few
+simple lines of code to give your UI a **fresh, modern and highly
+interactive** look.
 
-Before
+  - **shiny**
 
-</h2>
+<img src="man/figures/ss_before.png" alt="shiny" align="right" width="40%" />
 
-![Before](man/figures/ss_before.png)
-
-<h2>
+``` r
+library(shiny)
+ui <- fluidPage(
+  div(
+    div(
+      a("Link"),
+      p("Lorem ipsum, lorem ipsum, lorem ipsum"),
+      actionButton("button", "Click")
+    )
+  )
+)
+```
 
-After
+  - **shiny.semantic**
 
-</h2>
+<img src="man/figures/ss_after.png" alt="semantic" align="right" width="40%" />
 
-![After](man/figures/ss_after.png)
+``` r
+library(shiny.semantic)
+ui <- semanticPage(
+  div(class = "ui raised segment",
+    div(
+      a(class="ui green ribbon label", "Link"),
+      p("Lorem ipsum, lorem ipsum, lorem ipsum"),
+      actionButton("button", "Click")
+    )
+  )
+)
+```
 
 ## Component examples
 
-![Components](man/figures/semantic_components.png)
-
-<p style="text-align: center; font-size: x-large;">
+<img src="man/figures/semantic_components.png" alt="" width="80%" />
 
-<a href="https://demo.appsilon.ai/semantic/">Components live demo</a>
-
-</p>
+<center>
 
-The source code for **Components live demo** is located in **/examples**
-folder. To run it locally you will have to install:
+<h3>
 
-  - [**highlighter**](https://github.com/Appsilon/highlighter)
+<a href="https://demo.appsilon.ai/semantic/">Components live demo</a>
 
-<!-- end list -->
+</h4>
 
-    devtools::install_github("Appsilon/highlighter")
+</center>
 
 Check out also our dashboard examples made with **shiny.semantic**
 library:
 
-1.  [Churn analytics](https://demo.appsilon.ai/churn)
-2.  [Fraud detection](https://demo.appsilon.ai/frauds)
+| [Churn analytics](https://demo.appsilon.ai/churn) | [Fraud detection](https://demo.appsilon.ai/frauds) |
```

Comment:
Ideally we have this in a for that Filip mentioned on the call so app on the left and code on the right with some comments: https://shiny.rstudio.com/ (see Here is a Shiny app)
but in the meantime can we provide links to apps source code?

Reply:
it can be done like this `runApp("examples/calendar/", display.mode = "showcase")`

but currently it does not look the greatest:
<img width="936" alt="Screenshot 2020-06-25 at 10 17 42" src="https://user-images.githubusercontent.com/4547289/85692301-327fab00-b6cd-11ea-928e-d88c8089dd09.png">

probably fomantic - boostrap incompatibility - requires detailed investigation... I added an issue for that: https://github.com/Appsilon/shiny.semantic/issues/202